tests/spec/core/xref-spec.js

```diff
@@ -418,4 +456,113 @@ describe("Core — xref", () => {
       "[html],[infra],[local-1],[local-2],[local-3],[local-4],[webidl]"
     );
   });
+
+  describe("inline IDL references", () => {
+    it("ignores inlines starting with backslash", async () => {
+      const body = `<section><p id="test">{{{\\PASS }}}</p></section>`;
```

Comment:
Does the space after PASS matter?

Reply:
It doesn't matter. I kept it to show it doesn't matter.